src/Cli.Tests/TestHelper.cs

```diff
@@ -41,6 +44,7 @@ public static Process ExecuteDabCommand(string command, string flags)
                 StartInfo =
                 {
                     FileName = @"./Microsoft.DataApiBuilder",
+                    CreateNoWindow = true,
```

Comment:
Does this mitigate the issues seen in the previously ignored test in EnvironmentTests caused by hangs?

Reply:
Yes. seems like it, when I ran the test locally, I noticed VS debug window popping up to get user input for debugging the failed start up of engine in the child process(- the failure is what the test expects). I had to click cancel and seems like in the pipeline it may have been hanging waiting for a similar user input. After adding this change, looks like the hang is gone from the pipeline so that confirms this fixed the issue.